packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [46.0.1]
+
+### Changed
+
+- Bump `@metamask/keyring-api` from `^13.0.0` to `^14.0.0` ([#5177](https://github.com/MetaMask/core/pull/5177))
+- Bump `@metamask/keyring-internal-api` from `^2.0.0` to `^2.0.1` ([#5177](https://github.com/MetaMask/core/pull/5177))
+- Bump `@metamask/keyring-snap-client` from `^2.0.0` to `^3.0.0` ([#5177](https://github.com/MetaMask/core/pull/5177))
+
+### Fixed
+
+- Fix type issue in `ERC721Standard.getDetails` ([#4985](https://github.com/MetaMask/core/pull/4985))
+  - The returned `image` variable should have a been a `string` when the token image was using the `'ipfs://'` protocol.
+- Relax NFTs metadata RPC calls ([#5134](https://github.com/MetaMask/core/pull/5134))
+  - We now check the number of NFTs to update against a threshold value (500) to avoid sending an excessive amount of RPC calls to fetch NFTs metadata.
```

Comment:
No sure if this is one is truly a "Fixed" (the PR was marked as such)

Yes this PR fixes an issue we had with excessive network calls, but it also now changes the behavior when fetching NFT metadata.

Should this be a "Changed" instead, also should we make this a minor update in that case?

cc: @sahar-fehri

Reply:
I think its fine keeping it as "Fixed". There were multiple updates to the pref controller that ended up triggering that function which shouldn't have been the case. I think the main issue was that we were not checking which params got updated and we were triggering the updateNftMetadata function regardless.

The 500 threshold we added was another "safe net" to avoid making the excessive amount of RPC calls.